### PR TITLE
feat: add schema for 2.6.0 and pulsar binding to ObjectBinding

### PR DESCRIPTION
**Pull request by VisualBean**
**Description**

- Added definitions for 2.6.0 in preparation for 2.6.0
- Built schema 2.6.0 in preparation for 2.6.0
- Added googlepubsub and pulsar to the BindingObject definition

**Related issue(s)**
[Pulsar binding PR](https://github.com/asyncapi/bindings/pull/173)
[Add pulsar to specification PR](https://github.com/asyncapi/spec/pull/882)

### CODE REVIEW

**Comment by fmvilas** [comment]
Hey @VisualBean! The file you submitted is an invalid JSON and it has double the lines as the previous one. It looks like you pasted the new one after the old one 😅 Please have a look again.

**Comment by VisualBean** [comment]
> Hey @VisualBean! The file you submitted is an invalid JSON and it has double the lines as the previous one. It looks like you pasted the new one after the old one 😅 Please have a look again.

Dont copy/paste, late in the night.

**Comment by fmvilas** [comment]
Sorry once again. Forgot that recently we refactored the code in this repo so these files are generated automatically. You actually have to change this file: https://github.com/asyncapi/spec-json-schemas/blob/84c8d5d0831407d3d90f9d67c0556a18c7f56600/definitions/2.5.0/bindingsObject.json. Well, not exactly this one, but a copy of it that you should place under the `definitions/2.6.0` folder.

After that, if you run `npm run build`, it should generate the file you were trying to modify.

**Comment by VisualBean** [comment]
> Sorry once again. Forgot that recently we refactored the code in this repo so these files are generated automatically. You actually have to change this file: https://github.com/asyncapi/spec-json-schemas/blob/84c8d5d0831407d3d90f9d67c0556a18c7f56600/definitions/2.5.0/bindingsObject.json. Well, not exactly this one, but a copy of it that you should place under the `definitions/2.6.0` folder.
> 
> After that, if you run `npm run build`, it should generate the file you were trying to modify.

There we go. - I didnt copy over the rest of the files though - Should i do this in preparation for 2.6.0?  - im not completely sure what changed though.

File: schemas/2.6.0.json [hunk 1]
```diff
@@ -0,0 +1,1847 @@
...
+                "nats": {},
+                "jms": {},
+                "sns": {},
+                "sqs": {},
+                "stomp": {},
+                "redis": {},
+                "ibmmq": {},
+                "solace": {},
+                "googlepubsub": {},
+                "pulsar": {}
```
Comment (VisualBean) [inline schemas/2.6.0.json:314]: The effective change is here

File: definitions/2.6.0/bindingsObject.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "nats": {},
+    "jms": {},
+    "sns": {},
+    "sqs": {},
+    "stomp": {},
+    "redis": {},
+    "ibmmq": {},
+    "solace": {},
+    "googlepubsub": {},
+    "pulsar": {}
```
Comment (VisualBean) [inline definitions/2.6.0/bindingsObject.json:22]: effective change is here

**Review by fmvilas — APPROVED** [review]
LGTM 👍 Do you mind having a look @KhudaDad414 @char0n @derberg @dalelane?

**Comment by philCryoport** [comment]
Hi all, just a question as spec `2.6.0` isn't on the high-level roadmap.  Any concept of when this will go live?

**Comment by fmvilas** [comment]
@philCryoport the release will happen this month. We have 4 release months: January, April, June, and September. Check out [this document](https://github.com/asyncapi/spec/blob/master/RELEASE_PROCESS.md#release-cadence).

**Review by KhudaDad414 — DISMISSED** [review]
LGTM!

**Comment by philCryoport** [comment]
> @philCryoport the release will happen this month. We have 4 release months: January, April, June, and September. Check out [this document](https://github.com/asyncapi/spec/blob/master/RELEASE_PROCESS.md?rgh-link-date=2023-01-11T05%3A27%3A53Z#release-cadence).

Thank you @fmvilas !  I didn't know to go look for that file.

**Review by KhudaDad414 — APPROVED** [review]

**Review by fmvilas — APPROVED** [review]

**Review by dalelane — APPROVED** [review]

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
because of https://github.com/asyncapi/spec-json-schemas/pull/302 we have conflicts here

**Comment by derberg** [comment]
conflict solved. It was related to a fact that in 2.5 apparently we did not add `googlepubsub` binding to proper json schema, but just to generated one https://github.com/asyncapi/spec-json-schemas/pull/253 and the other PR from @AceTheCreator to `next-spec` fixed it and added `googlepubsub` to the list, thus conflict. I hope that makes sense 😅

**Pull request merged**